app/lib/domains.server.ts

```diff
@@ -0,0 +1,146 @@
+import * as dns from './dns.server';
+import { RecordStatus } from '@prisma/client';
+import { prisma } from '~/db.server';
+import type { RecordType } from '@prisma/client';
+
+interface DnsRecord {
+  type: RecordType;
+  name: string;
+  value: string;
+}
+
+interface Record {
+  id?: number;
```

Comment:
I would use values from the `Record` type that Prisma has, to match them: `Record['id']` etc.

Reply:
Will be updated